lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -94,6 +95,12 @@ def self.increase_version(current_version, type_of_bump, snapshot)
         end
       end
 
+      private_class_method def self.latest_non_prerelease_version_number
+        Actions
+          .sh("git describe --tags $(git rev-list --exclude=\"*[a-zA-Z]*\" --tags=\"[0-9]*.[0-9]*.*[0-9]\" --max-count=1)")
```

Comment:
This is a bit messy... But the `exclude` and `tags` options in the `git rev-list` command accept a `glob` and not a `regex` which was making filtering out unwanted tags difficult. In the end, the current behavior will be that it will filter out tags containing any letters, and include tags that match this glob `[0-9]*.[0-9]*.*[0-9]`. Lmk if you know of a better way to do this!

Reply:
I think the only alternative would be to do it programmatically using ruby and calculating the highest tag?

Reply:
Yeah that's the best alternative... I wasn't sure about doing that since the list of tags can grow... But honestly, I think it's not going to be a problem. And I think it's going to be cleaner so I think I will do that now.

Reply:
I did the change. Lmk what you think @vegaro